css/styles.css

```
@@ -1,3 +1,234 @@
+.visuallyhidden {
+	position: absolute;
+	clip: rect(0 0 0 0);
+	height: 1px;
+	width: 1px;
+	overflow: hidden;
+	padding: 0;
+	border: 0;
+}
+
+.visuallyhidden:focus {
+	clip: auto;
+	height: auto;
+	width: auto;
+}
+
+.bct{
+	display: flex;
+	font-size: 94%;
+}
+
+.cr{
+	padding: 0 0.83em;
+	line-height: 1.5;
+}
+
+.tag:not(body).is-danger{
+	background-color: #EB0000 !important;
+}
+
+.button.is-primary.is-large{
+	width: 100%;
+}
+
+/* *:focus {
+	outline: 2px dashed red!important;
+	border-radius: 2px;4
+} */		
+button:focus,
+input[type=checkbox]:focus,
+input[type=radio]:focus,
+input[type=submit]:focus {
+	box-shadow: 0 0 0 3px #ffffff, 0 0 0 6px #1274C4 !important;
+	outline: none;
+}
+.skip-to{
```

I suggest adding styles like this

```suggestion
.skip-to a {
	padding: 6px;
	position: absolute;
	top: -40px;
	left: 0px;
	color: white;
	border-right: 1px solid white;
	border-bottom: 1px solid white;
	border-bottom-right-radius: 8px;
	background: #BF1722;
	-webkit-transition: top 1s ease-out;
	transition: top 1s ease-out;
	z-index: 100;
}

.skip-to a:focus {
	position: absolute;
	left: 0px;
	top: 0px;
	outline-color: transparent;
	transition: top .1s ease-in;
}
```

---

Oh, I didn’t notice that I have an extra class without properties there. On lines 81-94 I already described the same behavior of the link class skip-link